app/helpers/outputs_helper.rb

```diff
@@ -1,11 +1,11 @@
 # frozen_string_literal: true
 
 module OutputsHelper
-  def link_to_error_line script, error, valid
+  def link_to_error_line object, error, valid = true, extra_params = {}
```

Comment:
La última y no jodo más (al menos con este método). Me parece mejor si lo dejamos así:

```suggestion
  def link_to_error_line object, error, valid: true, extra_params = {}
```

Porque sino cuando lo llamamos tenemos que andar viendo el helper para ver que carajo era eso =)

Reply:
d1 queda mas entendible :+1: